src/omitBy.js

```diff
@@ -0,0 +1,25 @@
+var _curry2 = require('./internal/_curry2');
+var complement = require('./complement');
+var pickBy = require('./pickBy');
+
+/**
+ * Returns a partial copy of an object omitting the keys that
+ * satisfy the supplied predicate.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @sig [String] -> {String: *} -> {String: *}
```

Comment:
The first argument should be something like `((a, String) -> Boolean)` rather than `[String]`.